bool picking
string[] boxA
string[] boxB
string[] boxC
string[] boxD
string[] boxE
string[] boxF
string[] boxG
string[] boxH
string[] boxI
string[] boxJ
string[] boxK
string[] boxL
string[] tote

WorkingItem[] workingItems

int8 currentWorkItem

bool attempt_running
time attemped_start_time

